document-service/documents-ui/src/components/documentIndexingForm/index.vue

```diff
@@ -47,6 +40,38 @@ const hasDescriptionError = computed(() => {
   return description.value.length > 1000
 })
 
+const docIdTrailingIcon = computed(() => {
+  if (docIdError.value !== '' || consumerDocumentId.value.length === 0) return ""
+  if (isDocIdLoading.value) return "i-mdi-loading"
+  return "i-mdi-check"
+})
+
+/** Watch the document ID for validation and update the error message accordingly. */
+watch(() => consumerDocumentId.value.trim(), async (docId: string) => {
+  if (docId && !/^\d+$/.test(docId)) {
+    docIdError.value = "Must contain numbers only"
+    return
+  }
+  if (docId.length < 8 && !noDocIdCheckbox.value) {
+    docIdError.value = "Enter the 8-digit Document ID number, also referred to as the barcode number"
+    return
+  } 
+  docIdError.value = ""
+  isDocIdLoading.value = true
+  const response = await getDocumentRecord(docId)
```

Comment:
I believe there was a dedicated endpoint for the Doc ID verification. 

https://okagqp-test-bcregrestricted.apigee.io/docs/docserviceproxy/1/routes/doc/api/v1/documents/verify/%7BconsumerDocumentId%7D/get

Reply:
Oh yes, I noticed that you replaced `docAPI/verify` with `docAPI/search`.
I will address it shortly. 😄

Reply:
Perfect, thank you!